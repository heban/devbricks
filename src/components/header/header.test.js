import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Header from './header';

it('renders correctly', () => {
  const component = shallow(<Header />);

  expect(toJson(component)).toMatchSnapshot();
});

it('contains <h1> tag', () => {
  const component = shallow(<Header />);

  expect(component.find('h1')).toHaveLength(1);
});
