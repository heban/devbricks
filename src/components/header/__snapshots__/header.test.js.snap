// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header__HeaderWrapper>
  <h1>
    DevBricks
  </h1>
</header__HeaderWrapper>
`;
